src/platform/site-wide/sass/shame.scss

```diff
@@ -589,3 +589,13 @@ img[data-srcset] {
     width: 100%;
   }
 }
+
+.campaign-landing-page {
+  .stories {
+    .story-teaser {
+      img {
+        max-height: 150px;
+      }
+    }
+  }
+}
```

Comment:
Following the pattern set by the VAMC team, can we add a dedicated SCSS module for this layout into `src/applications/static-pages/sass/modules` which we can import from `src/applications/static-pages/sass/static-pages.scss`? Then that whole module can belong to styles for this layout and associated components.

Instead of nesting selectors, I'm wondering if we can flatten this out by using CSS namespaces or the BEM syntax. It can make refactoring a zillion times easier if/when the DOM changes. 

```scss
// src/applications/static-pages/sass/modules/_m-campaign-landing-page-layout.scss

// "l" for layout like we have "va-l-detail-page"
.va-l-campaign-landing-page {
   // styles specific to the whole page layout itself
}

.va-c-stories {
  // Maybe this would be fine 
}

// Maybe we have a more descriptive name? Maybe "va-c-campaign-teaser"?
.va-c-story-teaser {

}

// Add this class directly to the <img/> so that the CSS isn't so dependent on the shape of the DOM
.va-c-story-teaser-image {
  max-height: 150px;
}
```